src/index.js

```diff
@@ -142,7 +147,149 @@ const submitFormButton = document.getElementById('submitForm');
 const addEthereumChain = document.getElementById('addEthereumChain');
 const switchEthereumChain = document.getElementById('switchEthereumChain');
 
+// Library switch feature
+let libraryInUse = 'ethers';
+const radioEthers = document.getElementById('radio-ethers');
+const radioWeb3 = document.getElementById('radio-web3');
+
+// Deployed piggy bank smart contract
+let piggyBankContract;
+let ethersPiggyBankContract;
+let web3PiggyBankContract;
+
+// Deployed hst smart contract
+let hstContract;
+let ethersHstContract;
+let web3HstContract;
+
+// Deployed collectibles smart contract
+let collectiblesContract;
+let ethersCollectiblesContract;
+let web3CollectiblesContract;
+
+// Deployed failing smart contract
+let failingContract;
+let ethersFailingContract;
+let web3FailingContract;
+
+// Smart contract buttons state
+radioEthers.addEventListener('click', (event) => {
+  handleLibraryChange(event);
+});
+
+radioWeb3.addEventListener('click', (event) => {
+  handleLibraryChange(event);
+});
+
+function initializeLibrarySwitches() {
+  if (libraryInUse === 'ethers') {
+    radioEthers.checked = true;
+  } else {
+    radioWeb3.checked = true;
+  }
+}
+
+function handleLibraryChange(event) {
+  libraryInUse = event.target.value;
+  if (libraryInUse === 'web3') {
+    // Deployed piggy bank smart contract with web3 library
+    piggyBankContract = web3PiggyBankContract;
+    // Deployed hst smart contract with web3 library
+    hstContract = web3HstContract;
+    // Deployed collectibles smart contract with web3 library
+    collectiblesContract = web3CollectiblesContract;
+    // Deployed failing smart contract with web3 library
+    failingContract = web3FailingContract;
+  } else {
+    // Deployed piggy bank smart contract with ethers library
+    piggyBankContract = ethersPiggyBankContract;
+    // Deployed hst smart contract with ethers library
+    hstContract = ethersHstContract;
+    // Deployed collectibles smart contract with ethers library
+    collectiblesContract = ethersCollectiblesContract;
+    // Deployed failing smart contract with ethers library
+    failingContract = ethersFailingContract;
+  }
+
+  // Changing buttons and label state when piggy bank contract is deployed
+  if (piggyBankContract) {
+    contractStatus.innerHTML = 'Deployed';
+    depositButton.disabled = false;
+    withdrawButton.disabled = false;
+  } else {
+    contractStatus.innerHTML = 'Not clicked';
```

Comment:
This should probably say something like "Not deployed" instead of "Not clicked".

Reply:
Fixed in: 91555c04c8088ee472c87cc25843642994ec92b4